engine.py

```diff
@@ -402,13 +434,17 @@ def _get_dialog_parent(self):
         # Find a parent for the dialog - this is the Maya mainWindow()
         from tank.platform.qt import QtGui
         import maya.OpenMayaUI as OpenMayaUI
-        import shiboken
+
+        try:
+            import shiboken
+        except ImportError:
+            import shiboken2 as shiboken
```

Comment:
Should the ordering here match the PySide ordering we use?  Just imagining the possible scenario where PySide2 and PySide are both here (for whatever reason, and who knows if that would break)... we'd use PySide2 in one case and shiboken (rather than shiboken2) in the other.